tests/core.js

```diff
@@ -33,10 +33,10 @@ require([
             util.init({
                 core: {
                     onReady: function () {
-                        var dup;
                         util.addQuestion("Text", "question1");
-                        dup = util.addQuestion("Text", "question2");
-                        dup.p.nodeID = "question1";
+                        util.addQuestion("Text", "question2");
+                        var target = $("[name='property-nodeID']");
+                        target.val('question1').change();
```

Comment:
Hmm. That does seem to be working now. Now I'm not sure why I came to that conclusion anymore